### DISCUSSION TITLE
New Exploit Found.

### DISCUSSION THREAD

**Discussion by Charlieis1nsan3** [3kh0/ext-remover#1258] [Polls JOIN TITANIUM NETWORK TO SHARE PLEASE]
This is a exploit discovered by me you can view this in my Exploits-N-Shit repository at the bottom D3mstr0y. as of v130 and my board name dedede it works. I'm not sure for you guys. but could this possibly be a new skiovox exploit? Did this work for you?This exploit uses the Chromebook Demo mode.

**Comment by ghost** [comment]
upload this in the titanium network discord

**Reply by Charlieis1nsan3** [reply]
did it work for you?  😀

**Reply by buddyted** [reply]
what repository could I find it at

**Reply by schoolexploitkid** [reply]
https://github.com/Charlieis1nsan3/Exploits-n-shit

**Comment by Charlieis1nsan3** [comment]
bruh how does this not work for 66% -_-

**Reply by HelloIloveGimkit2** [reply]
bro no one's gonna use your shitty exploit if there's a better one around I mean like there's no point just use [br1ck](https://br1ck.vercel.app)

**Reply by py660** [reply]
> bro no one's gonna use your shitty exploit if there's a better one around I mean like there's no point just use [br1ck](https://br1ck.vercel.app)

At least having two methods is better than one; would you rather have only one method in this entire repository, since it "should cover all use cases"?

**Reply by py660** [reply]
nvm this is just the most basic powerwash tutorial, i stand corrected

**Comment by no-name-6655** [comment]
This isn’t even an exploit it’s just a feature that your admin didn’t block?

**Reply by HelloIloveGimkit2** [reply]
exactly

**Comment by py660** [comment]
Isn't this just a powerwash tutorial?

**Comment by awsome-webdev** [comment]
he got cooked 😳

**Reply by Charlieis1nsan3** [reply]
me? this is an vulnerability is that better. And I didn't say anything back because then I probably get backlash. Im already banned in 8 discord servers "For being racist"

**Reply by awsome-webdev** [reply]
ok.

**Reply by Charlieis1nsan3** [reply]
k🙃

**Comment by ghost** [comment]
dang this got heated , chill y'all it just people talking about exploits

**Comment by py660** [comment]
Quote from @Charlieis1nsan3:
> Im already banned in 8 discord servers "For being racist"

That's good for you, but none of us needed to know that.

***

I assume that the active contributors on this repo are quite appreciative of your contribution to the ever-expanding collection of exploits. However, the only issue the community has found with the content of your suggestion is that it has already been discovered and documented prior to this discussion. While there may have been a few potential communication breakdowns from others, and the feedback left here could have been more constructive and respectful, the general idea that this "exploit" was already discovered is consistent across most comments under this discussion, and no offense or harm should be assumed from within this thread.

**Reply by Charlieis1nsan3** [reply]
what the hell has it been discovered ?????? Th

**Reply by funkymokey** [reply]
what did bro do 😭

**Reply by Charlieis1nsan3** [reply]
ong

**Reply by HelloIloveGimkit2** [reply]
you posted a fucking fake exploit and make a whole thread about it for no reason TLDR; HE POSTED AN EXPLOIT THAT DOESN'T WORK

**Reply by py660** [reply]
It's just a matter of miscommunication. There is absolutely no need to use all caps to respond, and even less of a reason to respond in the first place.

**Reply by Charlieis1nsan3** [reply]
> you posted a fucking fake exploit and make a whole thread about it for no reason TLDR; HE POSTED AN EXPLOIT THAT DOESN'T WORK

OMFG hop off my fucking dick you fucking bitch ass n**** if you don't like it don't fucking say anything

**Reply by awsome-webdev** [reply]
jeeeeeez i didn't mean to start a comment war.

**Reply by Charlieis1nsan3** [reply]
> jeeeeeez i didn't mean to start a comment war.

your not the problem. your alright. I was answering your question and then all of that happened

**Reply by awsome-webdev** [reply]
I just realized this guy has also started arguments in 2 other threads

**Reply by HelloIloveGimkit2** [reply]
> > you posted a fucking fake exploit and make a whole thread about it for no reason TLDR; HE POSTED AN EXPLOIT THAT DOESN'T WORK
> 
> OMFG hop off my fucking dick you fucking bitch ass nigga if you don't like it don't fucking say anything

uh oh he's mad he's gonna do something he's gonna punch the screen LOL

**Reply by funkymokey** [reply]
> > > you posted a fucking fake exploit and make a whole thread about it for no reason TLDR; HE POSTED AN EXPLOIT THAT DOESN'T WORK
> > 
> > 
> > OMFG hop off my fucking dick you fucking bitch ass nigga if you don't like it don't fucking say anything
> 
> uh oh he's mad he's gonna do something he's gonna punch the screen LOL

both of you are fucking weird and irresponsible, take this lame ass shit to discord dms you fucking e-losers

**Reply by Charlieis1nsan3** [reply]
> > > > you posted a fucking fake exploit and make a whole thread about it for no reason TLDR; HE POSTED AN EXPLOIT THAT DOESN'T WORK
> > > 
> > > 
> > > OMFG hop off my fucking dick you fucking bitch ass n**** if you don't like it don't fucking say anything
> > 
> > 
> > uh oh he's mad he's gonna do something he's gonna punch the screen LOL
> 
> both of you are fucking weird and irresponsible, take this lame ass shit to discord dms you fucking e-losers

n**** stfu nobody cares about you. and first of all I'm a chill dude but that stupid pussy ass bitch likes fucking pissing people off before sucking his fucking dick. his fucking 0.00000000000001m  defeater

**Reply by HelloIloveGimkit2** [reply]
nah fam with a pfp like that stupid ass

**Reply by awsome-webdev** [reply]
yall just need to let it go its been 3 days 💀

**Reply by Charlieis1nsan3** [reply]
> yall just need to let it go its been 3 days 💀

cant help it if he's retarded, I think he's mentally challenged. thats why he keeps coming back.

**Reply by awsome-webdev** [reply]
maybe....
